PZ/WebClient/Connector.py

```diff
@@ -0,0 +1,94 @@
+import requests
+import json
+
+from .models import *
+
+class Connector(object):
+    def __init__(self, url_adr):
+        self.url_adr = url_adr
+        self.response = None
+        self.json_data = None
+        self.payload = {}
+
+    def set_url(self, url_adr):
+        self.url_adr = url_adr
+
+    def set_request_args(self, payload):
+        self.payload = payload
+
+    def to_json(self):
+        return json.dumps(self, default=lambda o: o.__dict__, sort_keys=True, indent=4)
+
+    def get_resources(self):
```

Comment:
Add aguments: 
    get_resources(self, name, page, page_size):
In case adding params to url_adr:
    https://www.example.com/resources?name=MSI&page=1&page_size=20